packages/postcss-minify-font-values/src/__tests__/index.js

```diff
@@ -116,6 +158,55 @@ test(
     'h1{font-family:\\$42}'
 );
 
+test(
+    'should correctly escape special characters at the end',
+    processCSS,
```

Comment:
You can use `passthroughCSS` for any test where the output is the same as the input.